tensor.h

```diff
@@ -28,25 +44,62 @@ class Tensor {
         this->backOp = op;
     }
 
+    bool isConstz(const Tensor<T> &one) {
```

Comment:
Would it be better to make these isConstsz functions private?

nit: why have you added 'z' after isConst?

Reply:
Will remove these functions as they were just for debugging if "this" is a const Tensor or not.